tests/src/Interop/WinRT/NativeComponent/Component.Contracts.BindingProjectionsTesting.h

```diff
@@ -3,7 +3,7 @@
 // See the LICENSE file in the project root for more information.
 #pragma once
 
-#include "Component.Contracts.BindingProjectionsTesting.g.h"
+#include "Component/Contracts/BindingProjectionsTesting.g.h"
```

Comment:
What happened here?

Reply:
The `cppwinrt` tool on pre 10.0.17763.0 SDKs doesn't support the `-prefix` flag, which changes the generated file structure to be dotted names instead of nested directories. I removed the flag and made the change as part of working through step-by-step to get the tests building on CI (which is finally working on AzDO).